sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -586,6 +614,10 @@ public DigitalTwinsServiceVersion getServiceVersion() {
     }
 
     Mono<Response<Void>> deleteRelationshipWithResponse(String digitalTwinId, String relationshipId, DeleteRelationshipOptions options, Context context) {
+        if (context == null) {
```

Comment:
since these are package private methods, they can be called from elsewhere within the namespace so we do a null check and assign a default value to all of them.